prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/UpdateCommand.java

```diff
@@ -179,6 +179,12 @@ public static class ApplyCommand extends AbstractCommand {
         @CommandLine.Option(names = CliConstants.CANDIDATE_DIR, required = true)
         Path candidateDir;
 
+        @CommandLine.Option(names = CliConstants.REMOVE)
+        boolean remove;
+
+        @CommandLine.Option(names = CliConstants.SKIP_CONFLICTS)
```

Comment:
turns out this part is already implemented (as `yes` parameter). Sorry my mistake I forgot it was already done in a different issue. Could you remove the `skipConflicts` part please?